Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
```

Comment:
### [activerecord](https://rubyonrails.org)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 6.0.3.3 → 6.1.3

[_[change-log](https://github.com/rails/rails/blob/v6.1.3/activerecord/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.1.3/activerecord)_]

:tada: Patched vulnerabilities:

- [CVE-2021-22880](https://nvd.nist.gov/vuln/detail/CVE-2021-22880)
  **Possible DoS Vulnerability in Active Record PostgreSQL adapter**

  
  URL: https://groups.google.com/g/rubyonrails-security/c/ZzUqCh9vyhI

  There is a possible DoS vulnerability in the PostgreSQL adapter in Active Record. This vulnerability has been assigned the CVE identifier CVE-2021-22880.  Versions Affected:  >= 4.2.0 Not affected:       < 4.2.0 Fixed Versions:     6.1.2.1, 6.0.3.5, 5.2.4.5  Impact ------ Carefully crafted input can cause the input validation in the "money" type of the PostgreSQL adapter in Active Record to spend too much time in a regular expression, resulting in the potential for a DoS attack.  This only impacts Rails applications that are using PostgreSQL along with money type columns that take user input.  Workarounds ----------- In the case a patch can't be applied, the following monkey patch can be used in an initializer:  ``` module ActiveRecord   module ConnectionAdapters     module PostgreSQL       module OID # :nodoc:         class Money < Type::Decimal # :nodoc:           def cast_value(value)             return value unless ::String === value              value = value.sub(/^\((.+)\)$/, '-\1') # (4)             case value             when /^-?\D*+[\d,]+\.\d{2}$/  # (1)               value.gsub!(/[^-\d.]/, "")             when /^-?\D*+[\d.]+,\d{2}$/  # (2)               value.gsub!(/[^-\d,]/, "").sub!(/,/, ".")             end              super(value)           end         end       end     end   end end ```






<details>
<summary>Commits</summary>

A change of **5364** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.3.3...v6.1.3).

These are the first 10 commits:
- (8cefe5f) [Fix couple of typos [ci skip] (#40578)](https://github.com/rails/rails/commit/8cefe5fc0d41c4326e2ed89b6860051a4d37fed6)
- (45042bb) [Fix typos [ci skip]](https://github.com/rails/rails/commit/45042bb0eb0e8f89fa36da7a656560f85c69904f)
- (9363b2a) [Merge pull request #40568 from jonathanhefner/guide-associat…](https://github.com/rails/rails/commit/9363b2ab193f22e84eaca00fbda881458a3ea9c8)
- (a99a2e9) [Merge pull request #40569 from jonathanhefner/guide-active_r…](https://github.com/rails/rails/commit/a99a2e9e2be488388c2d19b9c669fb7655d3e43e)
- (9e91199) [Merge pull request #40566 from jonathanhefner/guide-active_r…](https://github.com/rails/rails/commit/9e91199cf85256a17c844529add580ebca47c433)
- (b65239c) [Merge pull request #40565 from jonathanhefner/activerecord-m…](https://github.com/rails/rails/commit/b65239c9450e744d26004ddd004886264ad974ad)
- (5611ded) [Merge pull request #40008 from santib/upgrading-to-image-pro…](https://github.com/rails/rails/commit/5611dedd105558793e0708935a2d1cfbe1bb8f04)
- (113aa0f) [Merge pull request #40411 from p8/guides/version-badge](https://github.com/rails/rails/commit/113aa0f7c8a0b86a3aadda8bfe646463d7851b1f)
- (34f228b) [Merge pull request #40556 from jonathanhefner/guide-active_r…](https://github.com/rails/rails/commit/34f228b6c08b4b80aa7e6add98604049f01f781f)
- (ed50909) [Merge pull request #40555 from jonathanhefner/guide-active_r…](https://github.com/rails/rails/commit/ed50909f9c0de23069e688f00fe227def86530cc)


</details>